<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRohstoffeAbziehen</class>
 <widget class="QDialog" name="DlgRohstoffeAbziehen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zutaten vom Lager abziehen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TableView" name="tableViewMalz">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier werden die im Rezept angegebenen Mengen &lt;span style=&quot; font-weight:700;&quot;&gt;Malze &lt;/span&gt;aus dem &lt;span style=&quot; font-weight:700;&quot;&gt;Lagerbestand&lt;/span&gt; abgezogen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="TableView" name="tableViewHopfen">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier werden die im Rezept angegebenen Mengen &lt;span style=&quot; font-weight:700;&quot;&gt;Hopfen &lt;/span&gt;aus dem &lt;span style=&quot; font-weight:700;&quot;&gt;Lagerbestand&lt;/span&gt; abgezogen.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Im Reiter Rezept konnte dabei zwischen Menge nach Rezept und tatsächlicher Menge entschieden werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="TableView" name="tableViewHefe">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier werden die im Rezept angegebenen Mengen &lt;span style=&quot; font-weight:700;&quot;&gt;Hefe &lt;/span&gt;aus dem &lt;span style=&quot; font-weight:700;&quot;&gt;Lagerbestand&lt;/span&gt; abgezogen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="TableView" name="tableViewWZ">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier werden die im Rezept angegebenen Mengen &lt;span style=&quot; font-weight:700;&quot;&gt;Zusätze &lt;/span&gt;aus dem &lt;span style=&quot; font-weight:700;&quot;&gt;Lagerbestand&lt;/span&gt; abgezogen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Hinweis: die Mengenangaben lassen sich editieren.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="PushButton" name="btnAbziehen">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mit Klick auf den Button &lt;span style=&quot; font-weight:600;&quot;&gt;Zutaten abziehen&lt;/span&gt; wird die im Rezept verwendete Rohstoffmenge aus der Rohstoffdatenbank abgezogen und der Sud endgültig als gebraut markiert. &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#fc0107;&quot;&gt;Achtung:&lt;/span&gt;&lt;span style=&quot; color:#000000;&quot;&gt; Entscheidet man sich keine Zutaten abzuziehen, wird der Sud trotzdem als gebraut markiert.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Wird der Sud als nicht gebraut zurückgesetzt, erscheint das Feld &lt;span style=&quot; font-weight:600;&quot;&gt;Zutaten zurückgeben&lt;/span&gt;. &lt;/p&gt;&lt;p&gt;Dies verhindert irrtümliche Mengenangaben in der Rohstoffliste, die zu viel abgezogen wurden. Trotzdem ist bei Funktionen, die den Modus eines Sudes &amp;quot;zurücksetzen&amp;quot;, Vorsicht angebracht.&lt;/p&gt;&lt;p&gt;Sollen keine Rohstoffe abgezogen werden, ignoriert man diesen Button und schließt das Pop-Up-Fenster. Daraufhin benötigt der Kleine Brauhelfer eine erneute Bestätigung.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Zutaten abziehen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hiermit wird das Dialogfeld Zutaten abziehen &lt;span style=&quot; font-weight:600;&quot;&gt;geschlossen&lt;/span&gt;.&lt;/p&gt;&lt;p&gt;Vorausgesetzt es wurde vorher &lt;span style=&quot; font-weight:600;&quot;&gt;kein&lt;/span&gt; Klick auf den Button &amp;quot;Zutaten abziehen&amp;quot; getätigt, werden keine Rohstoffe aus der Rohstoffdatenbank abgezogen.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#fc0107;&quot;&gt;Achtung: &lt;/span&gt;Wurde der Sud zurückgesetzt und vorher die Rohstoffe abgezogen, müssen die Zutaten nun zurückgegeben werden. Andernfalls kommt es zu einer Fehlberechnung der Rohstoffe.&lt;/p&gt;&lt;p&gt;Bei Funktionen, die den Modus eines Sudes &amp;quot;zurücksetzen&amp;quot;, ist grundsätzlich Vorsicht angebracht.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>widgets/pushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>widgets/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgRohstoffeAbziehen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgRohstoffeAbziehen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
